### PR TITLE
Add ORD Entities to CMP

### PR DESCRIPTION
**Pull request by DimitarPetrov**
**Description**

Add ORD Entities to CMP in order to use them in the ORD Aggregator.

Changes proposed in this pull request:
- Add ORD Package entity with CRUD Service and Repository layers
- Add ORD Product entity with CRUD Service and Repository layers
- Add ORD Vendor entity with CRUD Service and Repository layers
- Add ORD Tombstone entity with CRUD Service and Repository layers
- Add new ORD fields to:
    - API
    - Event
    - Bundle
    - Application

**Pull Request status**

- [x] Implementation
- [x] Unit tests
- [x] Integration tests
- [x] `chart/compass/values.yaml` is updated

### CODE REVIEW

**Review by desislavaa — APPROVED** [review]

**Review by alextargov — APPROVED** [review]

File: components/director/internal/domain/api/repository.go [hunk 1]
```diff
@@ -14,10 +14,16 @@ import (
 const apiDefTable string = `"public"."api_definitions"`
 
 var (
-	tenantColumn     = "tenant_id"
-	apiDefColumns    = []string{"id", "tenant_id", "bundle_id", "name", "description", "group_name", "target_url", "version_value", "version_deprecated", "version_deprecated_since", "version_for_removal"}
+	tenantColumn  = "tenant_id"
+	apiDefColumns = []string{"id", "tenant_id", "bundle_id", "package_id", "name", "description", "group_name", "target_url", "ord_id",
```
Comment (ataleksandrov) [inline components/director/internal/domain/api/repository.go:18]: just out of curiosity - do we need both **bundle_id** and **package_id**

Comment (DimitarPetrov) [reply components/director/internal/domain/api/repository.go:18]: API Resource has relation both to a package and a bundle. From ORD perspective those both are two different groupings with different semantics. Package = Documentation/Visualisation grouping and Bundle = Consumption Grouping.

**Review by desislavaa — APPROVED** [review]

**Review by alextargov — APPROVED** [review]

**Pull request merged**